### DISCUSSION TITLE
Project Description

### DISCUSSION THREAD

**Discussion by desbenn** [community/community#9358] [Projects and Issues] [answered]
I recommend that product description be added as one of the fields that can be set in the Settings section of Project Beta.

**Comment by aaronc** [comment]
Yes please 👍 . Especially helpful for outside contributors

**Comment by adrianbw** [comment]
Please please please. There's lots of context outside of GitHub to a project that I'd link to be able to link (designs, DoD, etc.).

**Comment by azenMatt** [comment, marked as answer]
We just added this - with a description and readme field - more details here - https://github.blog/changelog/2022-02-02-the-new-github-issues-02-02-update/

Let me know if this helps!

**Reply by desbenn** [reply]
Simply excellent! Keep up the good work!
